docs/tutorials/demo_data_science_project.ipynb

```diff
@@ -9,42 +9,67 @@
-   "id": "2a1b38a3-dd43-424e-be1a-9c731bdcf4f5",
+   "execution_count": 5,
    "metadata": {
-    "cell_marker": "\"\"\"",
     "tags": [
-     "remove_output"
+     "hide-output"
     ]
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: numpy in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (1.26.2)\n",
+      "Requirement already satisfied: pandas in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (2.1.3)\n",
+      "Requirement already satisfied: scikit-learn in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (1.3.2)\n",
+      "Requirement already satisfied: python-dateutil>=2.8.2 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from pandas) (2.8.2)\n",
+      "Requirement already satisfied: pytz>=2020.1 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from pandas) (2023.3.post1)\n",
+      "Requirement already satisfied: tzdata>=2022.1 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from pandas) (2023.3)\n",
+      "Requirement already satisfied: scipy>=1.5.0 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from scikit-learn) (1.11.4)\n",
+      "Requirement already satisfied: joblib>=1.1.1 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from scikit-learn) (1.3.2)\n",
+      "Requirement already satisfied: threadpoolctl>=2.0.0 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from scikit-learn) (3.2.0)\n",
+      "Requirement already satisfied: six>=1.5 in /Users/maxmynter/Desktop/appliedAI/lakefs/lakefs-spec/.venv/lib/python3.11/site-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)\n",
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip available: \u001b[0m\u001b[31;49m22.3.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m23.3.1\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install numpy pandas scikit-learn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "%pip install lakefs-spec numpy pandas scikit-learn"
+    "Also install the appropriate lakeFS-spec version. That is either the latest release from PyPI via `pip install lakefs-spec`, or the development version from GitHub via `pip install git+https://github.com/aai-institute/lakefs-spec.git`."
```

Comment:
a) Please strip cell outputs before checking in the notebook, and b) isn't the previous instruction preferable here? It works in the case of both a CI install of a checkout, and a normal PyPI install.

In this current form, the notebook will not install lakefs-spec anymore if the user skips through the comments, which is not ideal.